tutorials/containers/microk8s-clustering

```diff
@@ -0,0 +1,71 @@
+---
+id: raspberrypi-clustering-instructions
+summary: Create a raspberry pi cluser using microk8s
+categories: clusters
+tags: kubernetes, beginner, multipass, container, microk8s
+difficulty: 3
+status: draft
+published: 2019-09-13
+author: Ammar Naqvi <ammar.naqvi@canonical.com>, Rhys Davies <rhys.davies@canonical.com>
+feedback_url: https://github.com/ubuntu/microk8s/issues/
+
+---
+
+Canonical is dedicated to enhancing the way we build and deliver technology through our continued investment in innovation and the open-source community. Over the past year, we’ve worked hard to realize the next phase of scaling and deployments on Kubernetes; we introduced [MicroK8s][microk8s], the lightweight, fast, powerful Kubernetes up in 60 seconds.
+
+
+We’ve heard from our users, partners, and customers and today, we’re pleased and excited to announce multi-node clustering in MicroK8s. You can now join multiple deployments to form a cluster. Welcome to simple, enterprise-grade Kubernetes! 
+
+# Test-Drive Time
+
+A preview release is available for you to test and give us your feedback. Lets set up the MicroK8s cluster.
+On two or more machines install MicroK8s from the 1.15/edge/clustering channel:
```

Comment:
What needs to be running on the PIs? What are the required setup steps before being able to run this command? cf. missing requirements.

Reply:
This is a whole other tutorial I want to do as it's too many steps that aren't directly related to this tutorial. What's best, have brief instructions here, or wait and link?